std/numeric.d

```diff
@@ -1914,9 +1914,9 @@ do
         }
         {
             auto ret = findLocalMin!T((T x) => -fabs(x), -1, 1, T.min_normal, 2*T.epsilon);
-            assert(ret.x.fabs.approxEqual(T(1)));
-            assert(ret.y.fabs.approxEqual(T(1)));
-            assert(ret.error.approxEqual(T(0)));
+            assert(ret.x.fabs.isClose(T(1)));
+            assert(ret.y.fabs.isClose(T(1)));
+            assert(ret.error.isClose(T(0), 0.0, 100*T.epsilon));
```

Comment:
`T.epsilon` was too small here, so I increased this value by two magnitudes. Still better than `1e-2` which was used by `approxEqual`. But maybe this is a hint, that the implementation of `findLocalMin` could be improved.